exalearn-api/ExaLearn.Dal/Repositories/HistoryRepository.cs

```diff
@@ -9,5 +13,15 @@ public class HistoryRepository : GenericRepository<History>, IHistoryRepository
         public HistoryRepository(ExaLearnDbContext appDbContext) : base(appDbContext)
         {
         }
+
+        public async Task<IList<History>> GetUserHistoryByIdAsync(int userId)
+        {
+            return await _appDbContext.Histories.Include(x => x.User).Include(x => x.PassedTest).Where(x => x.UserId == userId).ToListAsync();
```

Comment:
it won't work. What kind of bind does exist between PassedTest and History Entities?

Reply:
There isn't bind between PassedTest and History Entities, I forget remove redundant code.